src/blenderbim/blenderbim/bim/module/georeference/operator.py

```diff
@@ -79,8 +72,7 @@ def execute(self, context):
             new.is_null = Data.map_conversion[attribute.name()] is None
             new.is_optional = attribute.optional()
             # Enforce a string data type to prevent data loss in single-precision Blender props
-            new.data_type = "string"
-            new.string_value = "" if new.is_null else str(Data.map_conversion[attribute.name()])
+            new.set_value("" if new.is_null else str(Data.map_conversion[attribute.name()]))
```

Comment:
See above comment - I would prefer to use `import_attributes`. Also triple check that the data type of string is enforced even though the detected data types are floats, since it is vital to prevent data loss.